### PR TITLE
update title and description

### PR DESCRIPTION
**Pull request by Blockchainbrett**

### CODE REVIEW

**Comment by Blockchainbrett** [comment]
Do I need to put a break in the description for the description to show more than one line on the google search? @giankotarola

**Comment by morochroyce** [comment]
The other title and description metatags should match

File: Public/index.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <html lang="en-US">
 <head>
 	<!-- Primary Meta Tags -->
-<title>Fleek</title>
+<title>Fleek: Build. Preview. Deploy. Scale.</title>
 <meta name="title" content="Fleek" />
```
Comment (giankotarola) [inline Public/index.html:6]: This content should be `Fleek: Build. Preview. Deploy. Scale.` too.

**Comment by Blockchainbrett** [comment]
https://github.com/FleekHQ/fleek-home/pull/12
i just made this other PR to update all of it but getting conflicts. @giankotarola can you resolve the conflicts and merge those please?

**Comment by giankotarola** [comment]
Closed in favor of #12

**Pull request closed without merging**